pruntime/enclave/src/contracts/web3analytics.rs

```diff
@@ -181,7 +199,9 @@ pub struct Web3Analytics {
 
     key: Vec<u8>,
     #[serde(skip)]
-    parser: woothee::parser::Parser
+    parser: woothee::parser::Parser,
+
+    configuration: BTreeMap<AccountIdWrapper, bool>
```

Comment:
```suggestion
    no_tracking: BTreeMap<AccountIdWrapper, bool>
```

Let's give it to a more meaningful name.